### PR TITLE
Add example with reuters dataset.

### LINKED ISSUE #1219: Public datasets list
This is a list of public datasets, that we need to ensure AutoKeras is having good performances on.
Welcome to post more.

| Name | API | Code | Metric | Results |
| - | - | - | - | - |
| [MNIST](http://yann.lecun.com/exdb/mnist/)  | ImageClassifier| [code](https://github.com/keras-team/autokeras/blob/master/benchmark/mnist.py) | Accuracy | 99.04% |
| [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html)   | ImageClassifier|[code](https://github.com/keras-team/autokeras/blob/master/benchmark/cifar10.py) |Accuracy | 97.10% |
| [CIFAR100](https://www.cs.toronto.edu/~kriz/cifar.html)  | ImageClassifier | | Accuracy |
| [IMDB](https://ai.stanford.edu/~amaas/data/sentiment/)  | TextClassifier |[code](https://github.com/keras-team/autokeras/blob/master/benchmark/imdb.py) | Accuracy | 93.93% |
| [Titanic](https://www.tensorflow.org/datasets/catalog/titanic)  | StructuredDataClassifier |  [code](https://github.com/keras-team/autokeras/blob/master/benchmark/imdb.py)| Accuracy | 82.20% |
| [Wine](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_wine.html#sklearn.datasets.load_wine)  | StructuredDataClassifier | | Accuracy|
| [Iris](https://archive.ics.uci.edu/ml/datasets/iris) | StructuredDataClassifier | | Accuracy |
| [California Housing](https://scikit-learn.org/stable/datasets/index.html#california-housing-dataset)  | StructuredDataRegression |  [code](https://github.com/keras-team/autokeras/blob/master/benchmark/housing.py) | MSE | 0.23
| [Houses](https://github.com/emanhamed/Houses-dataset)  | IO API | | MSE| |

### PR DESCRIPTION
**Pull request by toohsk**
### Which issue(s) does this Pull Request fix?

resolves #1219

### Details of the Pull Request

Add an example using the Reuters dataset.

### CODE REVIEW

**Review by haifeng-jin — APPROVED** [review]
Thanks for the PR! LGTM
